cmd/podman/system/dial_stdio.go

```diff
@@ -0,0 +1,156 @@
+package system
+
+import (
+	"context"
+	"io"
+	"net"
+	"net/http"
+	"os"
+
+	"github.com/containers/podman/v3/cmd/podman/registry"
+	"github.com/containers/podman/v3/cmd/podman/validate"
+	"github.com/containers/podman/v3/pkg/bindings"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	dialStdioCommand = &cobra.Command{
+		Use:    "dial-stdio",
+		Short:  "Proxy the stdio stream to the daemon connection. Should not be invoked manually.",
+		Args:   validate.NoArgs,
+		Hidden: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runDialStdio()
+		},
+		Example: "podman system dial-stdio",
+	}
+)
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Command: dialStdioCommand,
+		Parent:  systemCmd,
+	})
+}
+
+func runDialStdio() error {
+	ctx := registry.Context()
+	ctx, cancel := context.WithCancel(ctx)
+	defer cancel()
+	bindCtx, err := bindings.NewConnection(ctx, "unix:///var/run/docker.sock")
```

Comment:
Is this configurable at all?

Reply:
we should default to same as the podman remote connection, so `unix:///run/podman/podman.sock` and `unix:///run/user/<UID>/podman/podman.sock` as rootless
if they run a podman command we should talk to the podman socket and not docker.

Reply:
Alright, I've updated it to grab the URI from the config which handles it as you say (unix:///run/podman/podman.sock for root, unix:///run/user/<UID>/podman/podman.sock for rootless by default)